### PR TITLE
D8CORE-000 Update Past Event List

### PR DESCRIPTION
**Pull request by jdwjdwjdw**
# READY FOR REVIEW

# Summary
- Past Events List fixup
  - The title h3 was not rendering correctly
  - The subheadline was an h3 which conflicted with the title
Screenshot 2024-03-22 at 5 22 06 PM

After fixup
Screenshot 2024-03-22 at 5 37 07 PM

# Review By (Date)
- ASAP

# Urgency
- High

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Add a past event with subheadline. Add a basic page with a past event list. Confirm the past event title and subheadline look correct.
3. Review code 🌦️

### CODE REVIEW

File: config/sync/views.view.stanford_events.yml [hunk 1]
```diff
@@ -5261,7 +5261,7 @@ display:
           exclude: 0
           alter:
             alter_text: 1
-            text: "<a class=\"su-link{% if su_event_source %} su-link--external{% endif %}\" href=\"{% if su_event_source %}{{ su_event_source }}{% else %}{{ view_node }}{% endif %}\">\r\n  {{ title }}\r\n</a>"
+            text: "<h3>\r\n  <a class=\"su-link{% if su_event_source %} su-link--external{% endif %}\" href=\"{% if su_event_source %}{{ su_event_source }}{% else %}{{ view_node }}{% endif %}\">\r\n  {{ title }}\r\n  </a>\r\n</h3>"
```
Comment (pookmish) [inline config/sync/views.view.stanford_events.yml]: This change is unnecessary. You removed the `h3` from one place, to add it in another. Please revert this change. You should only be making changes to the subheadline field, not the title field.

**Review by pookmish — CHANGES_REQUESTED** [review]

File: config/sync/views.view.stanford_events.yml (hunk 1 above)
Comment (jdwjdwjdw) [reply config/sync/views.view.stanford_events.yml]: Do you see the screenshot and summary text I added at the top of this PR? When I spin up a new instance of Stanford Profile, it's not rending the title correctly unless I update it this way.

image

Comment (pookmish) [reply config/sync/views.view.stanford_events.yml]: If you provide steps how to reproduce the issue, I'd be able to review. But at this time, the title field has the same markup.

Comment (jdwjdwjdw) [reply config/sync/views.view.stanford_events.yml]: 1. Open a new gitpod workspace for https://github.com/SU-SWS/stanford_profile
2. Refer to Step 2 under `Setup tasks and/or behavior to test` and confirm it looks incorrect

Comment (pookmish) [reply config/sync/views.view.stanford_events.yml]: I fixed it myself. see the commit: https://github.com/SU-SWS/stanford_profile/pull/783/commits/b9671b58f30aa47b6f851f989bbc3ab2c64b8dd8
